extensions/_beacons/http_status.py

```diff
@@ -127,9 +131,6 @@ def beacon(config):
                                'path': attr_path
                                }
                     ret.append(_failed)
-            else:
```

Comment:
The full `if` and `else` block should get moved to the validate function, rather than waiting until the check to verify if the comparison is supported.